### DISCUSSION TITLE
border_width not accounted for when using -m -2

### DISCUSSION THREAD

**Discussion by ortango** [davatorium/rofi#1968] [(Possible) Issues] [answered]
hey, i would have filed a normal issue, but this is simple enough to post here:

shouldn't border_width be part of the window pos/size calculation when overlaying rofi on the focused window? (`-m -2`)? i am using bspwm for reference.

i see two choices (in xcb.c:monitor_active_from_id_focused):

include border

      mon->w = r->width + (r->border_width * 2);
      mon->h = r->height + (r->border_width * 2);

or exclude border

      mon->x = t->dst_x - r->x + r->border_width;
      mon->y = t->dst_y - r->y + r->border_width;

i am using the second one, it seems more correct and fits my usage.

**Comment by DaveDavenport** [comment, marked as answer]
I don't use this option myself, so I don't really have an opinion on this.
Could you make a PR that implements and documents the option  you think is the most natural?

Thanks!

(giving it some short thoughts, I would say I agree with you and would go for the 2nd one).

**Comment by ortango** [comment]
Absolutely, i'll make a PR in a little while. Thanks for taking a look.
